openpype/hosts/resolve/api/menu.py

```diff
@@ -111,6 +119,15 @@ def __init__(self, *args, **kwargs):
         # reset_resolution_btn.clicked.connect(self.on_set_resolution_clicked)
         experimental_btn.clicked.connect(self.on_experimental_clicked)
 
+    def on_save_current_clicked(self):
+        host = registered_host()
+        current_file = host.current_file()
+        if not current_file:
```

Comment:
Maybe this check could be done in the UI building and disable the button altogether

Reply:
It's tricky that there's no callbacks coming from Resolve currently and the UI is intended to remain open over time - so it'd be hard to detect when to disable/enable the button unfortunately. It would definitely have been nicer :)

But if we have _save callbacks_ we could always then start sneaking in the "save current file" logic there so it also always exports to the project. Making the button redundant. But alas, we're not there yet.

Reply:
@jakubjezek001 could you try again to see if it's better now?